pylibgen/pylibgen.py

```diff
@@ -132,7 +132,7 @@ def lookup(
         if "id" not in fields:
             fields.append("id")
         if "*" in fields:
-            fields = ["*"]
+            fields = list(constants.ALL_BOOK_FIELDS)
```

Comment:
This isn't desirable as libgen's book schema is pretty volatile, and tests need to be able to detect when the fields returned by a `*` query changes. [Example](https://github.com/JoshuaRLi/pylibgen/commit/f616f8c2d8270b0fc50c52d60e216f815a8cbfe6#diff-a7cffebd4b9dc807b165a9d726aa040aL91).